packages/libs/components/src/plots/ExportPlotToImageButton.tsx

```diff
@@ -0,0 +1,55 @@
+import { Image } from '@material-ui/icons';
+import { FloatingButton } from '@veupathdb/coreui';
+
+interface ToImageOpts {
+  height: number;
+  width: number;
+  format: 'svg' | 'png';
+}
+
+interface ToImage {
+  (opts: ToImageOpts): Promise<string>;
+}
+
+interface Props {
+  filename?: string;
+  toImage: ToImage;
+}
+
+export function ExportPlotToImageButton(props: Props) {
+  const { filename = 'plot', toImage } = props;
+  const ext = 'svg';
+  return (
+    <div
+      style={{
+        width: '100%',
+        display: 'flex',
+        justifyContent: 'end',
+        padding: '.5em 0',
+      }}
+    >
+      <FloatingButton
+        onPress={() => downloadPng(toImage, filename, ext)}
+        text={`Export to ${ext.toUpperCase()}`}
+        icon={Image}
+        themeRole="primary"
+        textTransform="none"
+        size="small"
+      />
+    </div>
+  );
+}
+
+async function downloadPng(
```

Comment:
what do you think about calling this function something more general like `downloadImage`, since we also use it to download svgs?

Reply:
Yes, good catch! It's updated now